docs/versions/21.0.md

```diff
@@ -0,0 +1,25 @@
+# 21.0 Releases
+
+!!! note "We need your help"
+
+    Version 21.0.0 is the first significant update in HGVS Nomenclature in almost four years.
+    While we have resolved many errors during this work, it is possible that new errors were introduced. If you find an issue, please [contact us](/#contact-us).
+
+## 21.0.0 (ETA 2024-01-08)
+
+### Major Changes
+
+- **Clarified our name** The preferred name is now "HGVS Nomenclature". The many previous names (e.g., HGVS guidelines, HGVS recommendations, VarNomen, Mutnomen) created confusion and are obsolete.
+- **Chose a stable URL** To foster stable URLs and better serve the community, HGVS Nomenclature now has a dedicated domain name, [hgvs-nomenclature.org](hgvs-nomenclature.org). The previous pages are now archived at [archive.hgvs-nomenclature.org](https://archive.hgvs-nomenclature.org/). The old site, [varnomen.hgvs.org](https://varnomen.hgvs.org/) forwards users to the corresponding page on the new site.
+- **Modernized our documentation system** Pages from the old documentation were migrated to a system that should make it easier to maintain HGVS Nomenclature and support multiple device size (phones, tablets, laptops). The intention during migration was to retain the intention of the original text, while modernizing the management of HGVS Nomenclature and cleaning up the presentation.
+- **Switched versioning schemes** HGVS Nomenclature will now use Semantic Versioning, which should make it easier for users to track new features in HGVS Nomenclature. Beginning with this release, historical versions will also be available adjacent to newer versions. See [Versions](..) for details.
+- **Joined HUGO** HGVS Nomenclature is overseen by the [Human Variant Nomenclature Committee](/hvnc/), a subcommittee of the [Human Genome Organization (HUGO)](https://www.hugo-international.org/).
+- **Modernized mechanism to contact the HVNC** We have introduced two new mechanisms to contact the HVNC: there is now an email list to discuss HGVS nomenclature and we have formalized the process by which may submit ideas. See [Contact Us](/#contact-us) for details. The Facebook group and older email address (which was read only by Johan) are now obsolete.
```

Comment:
I would suggest changing "Johan" to "the committee's chair".

Reply:
Great idea.